suites/reef/rbd/tier-3_rbd_mirror_upgrade_scale.yaml

```diff
@@ -0,0 +1,193 @@
+#===============================================================================================
+# Tier-level: 3
+# Test-Suite: tier-3_rbd_mirror_upgrade_scale.yaml
+#
+# Cluster Configuration:
+#    cephci/conf/reef/rbd/5-node-2-clusters.yaml
+#    No of Clusters : 2
+#    Node 2 must to be a client node
+#===============================================================================================
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-rbd1:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    rhcs-version: 6.1
```

Comment:
wrong version, all comments applies to here as well.

Reply:
answered.